packages/react-components/src/mail.ts

```diff
@@ -4,12 +4,13 @@ interface MailOptions {
 }
 
 export const createMailTo = (
-  email: string,
+  emails: string | string[],
   { subject, body }: MailOptions = {},
 ): string => {
-  const mailTo = new URL(
-    `mailto:${email.split('@').map(encodeURIComponent).join('@')}`,
-  );
+  const mailTo =
+    typeof emails === 'string'
```

Comment:
I would just convert the string into an array with one entry. I would then use join so you don't have the trailing comma problem or require the other two functions

Reply:
That's a good idea